USAGE.md

```diff
@@ -198,9 +225,11 @@ $ tw launch my_rnaseq_nf_pipeline --params-file=my_rnaseq_nf_pipeline_params_2.y
     https://tower.nf/user/abhinav/watch/2XDXxX0vCX8xhx
 ```
 
-### Launching any pipeline
+See [Nextflow configuration][nextflow-config] for more information.
 
-Platform CLI can directly launch pipelines that have not been explicitly added to the Launchpad in a platform workspace by using the full pipeline repository URL:
+### Launch an unconfigured pipeline
```

Comment:
Is the wording "unconfigured pipeline" clear? I find it somehow confusing even though I know this means launching a run from and URL versus from a Pipeline in the platform.

I, also, don't have a better alternative to suggest

Reply:
Going with "unsaved pipeline". That sits better with me. WDYT?